modules/social_features/social_core/social_core.install

```diff
@@ -314,3 +314,34 @@ function social_core_update_123003(): void {
       ->save();
   }
 }
+
+/**
+ * Recreate views.view.message (module message) if it is corrupted.
+ */
+function social_core_update_123004(): void {
```

Comment:
We might want to add a dependency just to be sure that it runs after message 8107